src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -175,8 +175,8 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						}
 
 						value = MI.Get(video, i, "Format_Profile");
-						if (!value.isEmpty() && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H264)) {
-							media.setAvcLevel(getAvcLevel(value));
+						if (media.getCodecV() != null && isNotBlank(value)) {
```

Comment:
No.
I just tried to say that your appellation is more correct than the old one having nothing to do with AVC specific profiles, as all the video profiles are logged without any AVC filter.